src/test/java/core/basesyntax/services/actions/PurchaseHandlerTest.java

```diff
@@ -0,0 +1,51 @@
+package core.basesyntax.services.actions;
+
+import core.basesyntax.db.Storage;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class PurchaseHandlerTest {
+    private static Map<String, ActionHandler> actionHandlerMap;
+
+    @BeforeClass
+    public static void setUp() {
+        Storage.fruits.clear();
+        actionHandlerMap = new HashMap<>();
+        actionHandlerMap.put("p",new PurchaseHandler());
+    }
```

Comment:
```suggestion
    private static ActionHandler actionHandler;

    @BeforeClass
    public static void setUp() {
        Storage.fruits.clear();
        actionHandler = new PurchaseHandler();
    }
```
same for other handlers